src/applications/vaos/components/review/TypeOfCareSection.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function TypeOfCareSection(props) {
+  if (props.data.facilityType === 'communityCare')
+    return (
+      <h2 className="usa-alert-heading vads-u-padding-top--1">
+        Community care appointment
+      </h2>
+    );
+  else if (!props.data.facilityType !== 'communityCare')
```

Comment:
I don't think you need the `!` in front of `props`.

Reply:
Done